cardano-sync/src/Cardano/Sync/Era/Shelley/Generic/EpochUpdate.hs

```diff
@@ -7,11 +7,13 @@ module Cardano.Sync.Era.Shelley.Generic.EpochUpdate
   , shelleyEpochUpdate
   ) where
 
+import           Prelude hiding (Maybe)
+
 import           Cardano.Sync.Era.Shelley.Generic.ProtoParams
 import           Cardano.Sync.Era.Shelley.Generic.Rewards
 import           Cardano.Sync.Era.Shelley.Generic.StakeDist
 
-import           Data.Maybe (fromMaybe)
+import           Data.Strict.Maybe
```

Comment:
Imports from outside this repo should always be explicit or qualified. It makes it much more obvious where things come from.